generated/python/gapic-google-cloud-logging-v2/google/cloud/gapic/logging/v2/logging_service_v2_client.py

```diff
@@ -288,11 +315,11 @@ def write_log_entries(self,
             resource=resource,
             labels=labels,
             partial_success=partial_success)
-        self._write_log_entries(request, options)
+        return self._write_log_entries(request, options)
 
     def list_log_entries(self,
-                         project_ids,
-                         resource_names=None,
+                         resource_names,
+                         project_ids=None,
```

Comment:
Good thing we caught this before beta. Otherwise it would have been a breaking change to fix later.